R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -721,6 +721,17 @@ test_that(
   expect_true(dropTempView("dfView"))
 })
 
+test_that(
+  "createGlobalTempView() results in a queryable table and sql() results in a new DataFrame", {
+  df <- read.json(jsonPath)
+  createGlobalTempView(df, "global_table1")
+  newdf <- sql("SELECT * FROM global_temp.global_table1 where name = 'Michael'")
+  expect_is(newdf, "SparkDataFrame")
+  expect_equal(count(newdf), 1)
+  expect_true(dropGlobalTempView("global_table1"))
+  expect_false(dropGlobalTempView("global_table1"))
```

Comment:
Instead, I added the above line to check the success of deletion in the previous line. How do you think about this?